src/charts/bar.js

```diff
@@ -9,13 +9,13 @@ define(function(require) {
     const d3Format = require('d3-format');
     const d3Scale = require('d3-scale');
     const d3Selection = require('d3-selection');
-    const d3Transition = require('d3-transition');
```

Comment:
This needs to be loaded, even if we don't use it.

Reply:
Got it, i reversed it.

Reply:
Did you push the changes?

Reply:
No, did the changes locally only.